tools/reassign-alias.js

```diff
@@ -0,0 +1,43 @@
+const elastic = require('../lib/elastic')
+
+function displayErrorMessage (value) {
+  console.log(`You have not provided a value for ${value}. Please provide a value for ${value} with the --${value} option before continuing`)
+}
+
+function run (cluster, command) {
+  const { alias, oldIndex, newIndex } = command.opts()
+
+  if (!alias) {
+    displayErrorMessage('alias')
+  }
+  if (!oldIndex) {
+    displayErrorMessage('oldIndex')
+  }
+  if (!newIndex) {
+    displayErrorMessage('newIndex')
+  }
+
+  const client = elastic(cluster)
+  const clusterHost = global.workspace.clusters[cluster]
+
+  console.log(`Reassigning alias ${alias} from ${oldIndex} to ${newIndex} in ${cluster}: ${clusterHost}`)
+
+  const actions = {
+    'actions': [
+      { 'remove': { 'index': `${oldIndex}`, 'alias': `${alias}` } },
+      { 'add': { 'index': `${newIndex}`, 'alias': `${alias}` } }
+    ]
+  }
+
+  return client.indices.updateAliases({ body: { actions } })
```

Comment:
**Question:** If this errors will it output the error, I think it will but wanted to check

Reply:
Ooo I'm not sure! Good question. I think it will too... I'll test

Reply:
Put this into a try catch now so I think this comment is no longer relevant